src/vos/vos_pool.c

```diff
@@ -1030,38 +1033,49 @@ vos_pmemobj_open(const char *path, uuid_t pool_id, const char *layout, unsigned
 		goto umem_open;
 	}
 
+	DLCK_PRINT(dp, "PMEM pool... ");
 	/* No NVMe is configured or current xstream doesn't have NVMe context */
 	if (!bio_nvme_configured(SMD_DEV_TYPE_MAX) || xs_ctxt == NULL) {
+		DLCK_PRINT_YES_NO(dp, DLCK_YES);
 		store.store_type = DAOS_MD_PMEM;
 		goto umem_open;
 	}
+	DLCK_PRINT_YES_NO(dp, DLCK_NO);
 
-	D_DEBUG(DB_MGMT, "Open BIO meta context for xs:%p pool:"DF_UUID"\n",
-		xs_ctxt, DP_UUID(pool_id));
+	DLCK_PRINT(dp, "Open BIO meta context... ");
+	D_DEBUG(DB_MGMT, "Open BIO meta context for xs:%p pool:" DF_UUID "\n", xs_ctxt,
+		DP_UUID(pool_id));
```

Comment:
Can we warp these two lines as single call, such as `DLCK_DEBUG`, and inside `DLCK_DEBUG`, we can call `DLCK_PRINT` if available and then `D_DEBUG()`. That will make the main logic to be clear a bit. Similarly, the other `DLCK_PRINT` usage cases in the patch.